tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -210,9 +222,21 @@ namespace.ceph:
           enabled: true
           inputs:
             mandatory:
-              - Rbd.pool_id
               - Rbd.name
               - Rbd.size
+            optional:
```

Comment:
@gnehapk @anivargi They keys for the optional parameters are a little different than the existing keys ..  Just wanted to point that out 
`optional:
- Rbd.pool_id
- Rbd.pool_crush_ruleset
- Rbd.pool_erasure_code_profile
- Rbd.pool_min_size
- Rbd.pool_pg_num
- Rbd.pool_pgp_num
- Rbd.pool_poolname
 - Rbd.pool_type
- Rbd.pool_size
- Rbd.pool_quota_enabled
- Rbd.pool_quota_max_objects
- Rbd.pool_quota_max_bytes`